src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -479,10 +511,13 @@ case class IndexLogEntry(
     sourcePlanSignatures.head
   }
 
-  def hasLineageColumn(spark: SparkSession): Boolean = {
-    ResolverUtils
-      .resolve(spark, IndexConstants.DATA_FILE_NAME_COLUMN, schema.fieldNames)
-      .isDefined
+  def hasLineageColumn: Boolean = schema.fieldNames.contains(IndexConstants.DATA_FILE_NAME_ID)
```

Comment:
Can you revive `properties` under SparkPlan and keep this lineage config as @imback82 suggested previously?
I can make the change but it will conflict with your change. I think it's good to include any breaking change in this release.

Reply:
@sezruby  - I dont remember which suggestion you are referring. Can you confirm below is what you are asking here:

1. Bring back `properties` under `SparkPlan.Properties`:
```
object SparkPlan {
  case class Properties(
      relations: Seq[Relation],
      rawPlan: String,
      sql: String,
      fingerprint: LogicalPlanFingerprint,
      properties: Map[String, String] // <--- BRING BACK THIS
)	
}
```
2. If lineage is enabled for an index then store sth like `"hasLineage" -> "true"` in the Map.

Reply:
Yes and use the config in hasLineage function :)

Reply:
sure, it makes sense. I guess the only question is: 
Should we store it under `SparkPlan.properties.properties`? Because `SparkPlan` is under `Source`, however Lineage is really an index property, not a source property. Should not we add Lineage under `CoveringIndex.Properties` by adding a `properties: Map[String, String]` there?
```
object CoveringIndex {
  case class Properties(
             columns: Properties.Columns,
             schemaString: String,
             numBuckets: Int,
             properties: Map[String, String] // <--- ADD THIS
     )
  
object Properties {
    case class Columns(indexed: Seq[String], included: Seq[String])
  }
}
```

Reply:
Yep under Covering index looks better. Thanks!